### PR TITLE
Bug 1910127: ovirt: UPI doc improvements

### PR DESCRIPTION
**Pull request by dougsland**
Add the following docs changes:
- Several links to inventory.yml file
- Add documentation ref. to ovirt-config.yaml
- Note about python3
- Fixed TOC order
- Steps to ansible and ansible-galaxy
- Hint to CentOS users about epel-release repo and ansible

Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

### CODE REVIEW

File: docs/user/ovirt/install_upi.md [hunk 1]
```diff
@@ -43,16 +43,16 @@ mandatory ignition files and to monitor the installation process itself.
...
-## Prerequisites
-The `inventory.yml` file all the variables used by this installation can be customized as per
+# Prerequisites
+The [inventory.yml](../../../upi/ovirt/inventory.yml) file all the variables used by this installation can be customized as per
 user needs.
 The requirements for the UPI in terms of minimum resources are broadly the same as 
 the [IPI](./install_ipi.md#minimum-resources).
 
-- oVirt/RHV account stored in the `ovirt-config.yaml`
+- oVirt/RHV account stored in the [ovirt-config.yaml](https://github.com/openshift/installer/blob/6abb3b5a8b687ee38b6c96368c77a305f6f0b563/docs/user/ovirt/install_ipi.md#ovirt-configyaml)
```
Comment (janoszen) [inline docs/user/ovirt/install_upi.md:52]: Are you sure linking to this GitHub blob is a good idea? Shouldn't this link somewhere else? Shouldn't we make an example directory to contain examples?

File: docs/user/ovirt/install_upi.md [hunk 2]
```diff
@@ -69,11 +69,31 @@ the [IPI](./install_ipi.md#minimum-resources).
 To use the UPI process described here the following are required:
 
 - Python3
+
+```
+Currently most of Linux distros provides Python 3 by default.
```
Comment (janoszen) [inline docs/user/ovirt/install_upi.md:74]: I don't think abusing a code block like this is a good idea. I'd rather do it like this:

```
**Note:** Currently most of Linux distros provides Python 3 by default.
```

**Review by janoszen — COMMENTED** [review]
Hey @dougsland ,

thanks for working on the docs, they are one of my big personal <3. I have added a few comments, please take a look.

File: docs/user/ovirt/install_upi.md (hunk 2 above)
Comment (dougsland) [reply docs/user/ovirt/install_upi.md:74]: Kind of highlight to users but I am okay with this note.

File: docs/user/ovirt/install_upi.md (hunk 1 above)
Comment (dougsland) [reply docs/user/ovirt/install_upi.md:52]: It's a permanent link:
https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/getting-permanent-links-to-files

File: docs/user/ovirt/install_upi.md (hunk 2 above)
Comment (janoszen) [reply docs/user/ovirt/install_upi.md:74]: Yepp, but you are misusing an HTML element which will cause accessibility issues for folks who have visual disabilities.

File: docs/user/ovirt/install_upi.md (hunk 1 above)
Comment (janoszen) [reply docs/user/ovirt/install_upi.md:52]: Yes, but will it be kept updated? As far as I can tell this links to a file in a very specific commit. IMHO the link should read https://github.com/openshift/installer/blob/master/docs/user/ovirt/install_ipi.md#ovirt-configyaml

Comment (janoszen) [inline docs/user/ovirt/install_upi.md:52]: The link points to a specific commit and will be outdated. IMHO the link should read https://github.com/openshift/installer/blob/master/docs/user/ovirt/install_ipi.md#ovirt-configyaml

**Review by janoszen — APPROVED** [review]
LGTM

**Pull request merged**